app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss

```diff
@@ -97,7 +97,7 @@ $govuk-modal-wide-breakpoint: $govuk-page-width + $govuk-modal-margin * 2 + $gov
 }
 
 .gem-c-modal-dialogue__header {
-  padding: 8px govuk-spacing(3) 10px;
+  padding: 9px govuk-spacing(3) 10px;
```

Comment:
I don't really mind, but why have an explicit `px` value alongside a `govuk-spacing` value?

Reply:
Yeah, it's an odd mix isn't it. I can use pixels for all values for consistency. Tried to make sure the vertical rhythm is preserved, that's why I always use `govuk-spacing` for top and bottom.